src/Components/Highlight/Highlight.js

```diff
@@ -1,94 +1,109 @@
 import React, { Component } from 'react';
-import axios from 'axios';
 import Slider from 'react-slick';
 import {
 	HighlightContainer,
 	CollectionsContainer,
 	Image,
 	Container,
 	View,
-	Button,
+	Remove,
+	ImageOverlay,
+	Content,
+	AddBtn,
+	FormContainer,
+	SearchFormContainer,
 } from './Highlight.styles';
 import 'slick-carousel/slick/slick.css';
 import 'slick-carousel/slick/slick-theme.css';
 import HighlightContents from 'components/HighlightContents';
+import { ReactComponent as Add } from 'assets/add.svg';
+import SearchForm from 'components/SearchForm';
 
 export class Highlight extends Component {
 	state = {
-		collections: [],
-		isLoading: false,
-		hasError: false,
-		page: 1,
 		display: 'none',
+		formDisplay: 'none',
 		id: null,
 	};
+	form = React.createRef();
 
-	fetchNextPage = () => {
-		const nextPage = this.state.page + 1;
-		this.setState({ page: nextPage });
-	};
-
-	getPhoto = async () => {
-		try {
-			this.setState({ isLoading: true });
-			const { data } = await axios.get(
-				`https://api.unsplash.com/collections/?page=${this.state.page}&client_id=${process.env.REACT_APP_ACCESS_KEY}&order_by=latest`
-			);
-			this.setState({
-				collections: [...this.state.collections, ...data],
-				isLoading: false,
-			});
-		} catch (error) {
-			this.setState({ isLoading: false, hasError: true });
-			console.error(error);
-		}
-	};
 	showCollectionPhotos = (id) => {
 		this.setState({ display: 'block', id });
 	};
 	closeCollectionPhotos = () => {
-		this.setState({ display: 'none' });
+		this.setState({ display: 'none', formDisplay: 'none' });
 	};
 	componentDidUpdate(prevProps, prevState) {
-		if (this.state.page !== prevState.page) {
-			this.getPhoto();
+		if (
+			prevState.formDisplay !== this.state.formDisplay &&
+			this.state.formDisplay === 'block'
```

Comment:
because hre you are checking for true too, formDisplay === block => true | false